metrics.md

```diff
@@ -16,37 +16,58 @@ The metrics SHOULD behave under the guidelines set by the [Prometheus documentat
 
 ### Interop Metrics
 
-The following are the minimal metrics agreed to be conformed upon prior to initial client interop efforts.
+The following are the minimal metrics agreed to be conformed by the various client teams. 
 
 | Name | Metric type | Usage | Sample collection event |
-|------------------------------------------|-------------|-------------------------------------------------------------|----------------------|
-| `libp2p_peers`                           | Gauge       | Tracks number of libp2p peers                               | On peer add/drop     |
-| `beacon_slot`                            | Gauge       | Latest slot of the beacon chain state                       | On slot              |
-| `beacon_head_slot`                       | Gauge       | Slot of the head block of the beacon chain                  | On fork choice       |
-| `beacon_head_root`                       | Gauge       | Root of the head block of the beacon chain*                 | On fork choice       |
-| `beacon_finalized_epoch`                 | Gauge       | Current finalized epoch                                     | On epoch transition  |
-| `beacon_finalized_root`                  | Gauge       | Current finalized root*                                     | On epoch transition  |
-| `beacon_current_justified_epoch`         | Gauge       | Current justified epoch                                     | On epoch transition  |
-| `beacon_current_justified_root`          | Gauge       | Current justified root*                                     | On epoch transition  |
-| `beacon_previous_justified_epoch`        | Gauge       | Current previously justified epoch                          | On epoch transition  |
-| `beacon_previous_justified_root`         | Gauge       | Current previously justified root*                          | On epoch transition  |
+|--------------------------------------------|-------------|-------------------------------------------------------------|----------------------|
+| `libp2p_peer_connected_total`              | Gauge       | Tracks the total number of libp2p peers                     | On peer add/drop     |
```

Comment:
I would have thought given that 2 clients said `libp2p_peers` and the old standard was `libp2p_peers`, we could just leave this one as `libp2p_peers`.